app/src/routes/event/[slug]/+page.svelte

```diff
@@ -1,46 +1,52 @@
 <script lang="ts">
-	import { PortableText } from '@portabletext/svelte';
-	import { useQuery } from '@sanity/svelte-loader';
-	import { formatDate, formatTime } from '$lib/utils';
-	import { urlFor } from '$lib/sanity/image';
-	import type { PageData } from './$types';
+  import { PortableText } from "@portabletext/svelte";
+  import { useQuery } from "@sanity/svelte-loader";
+  import { formatDate, formatTime } from "$lib/utils";
+  import { urlFor } from "$lib/sanity/image";
+  import { signIn, signOut } from "@auth/sveltekit/client";
 
-	export let data: PageData;
-	const q = useQuery(data);
+  export let data;
 
-	$: ({ data: event } = $q);
+  const q = useQuery(data);
+  $: ({ data: event } = $q);
 </script>
 
 <section class="w-full mt-2 mb-8 mx-0">
-	{#if event.mainImage}
-		<img
-			class="w-full h-[380px] object-cover"
-			src={urlFor(event.mainImage).url()}
-			alt="Cover image for {event.title}"
-		/>
-	{/if}
-	<div class="pr-4 pl-3 py-0">
-		<h1 class="text-6xl font-extrabold leading-loose">{event.title}</h1>
-		{#if event.summary}
-			<p class="text-2xl">{event.summary}</p>
-		{/if}
-		<div class="my-6 font-bold">
-			<div class="flex">
-				<span class="material-icons mr-2">schedule</span>
-				<p>
-					{formatDate(event.start)}
-					{formatTime(event.start)}
-				</p>
-			</div>
-			<div class="flex">
-				<span class="material-icons mr-2">location_on</span>
-				<p> {event.place}</p>
-			</div>
-		</div>
-		{#if event.body}
-			<div class="mt-8 flex flex-col gap-4">
-				<PortableText components={{}} value={event.body} />
-			</div>
-		{/if}
-	</div>
+  {#if event.mainImage}
+    <img
+      class="w-full h-[380px] object-cover"
+      src={urlFor(event.mainImage).url()}
+      alt="Cover image for {event.title}"
+    />
+  {/if}
+  <div class="pr-4 pl-3 py-0">
+    <h1 class="text-6xl font-extrabold leading-loose">{event.title}</h1>
+    {#if event.summary}
+      <p class="text-2xl">{event.summary}</p>
+    {/if}
+    <div class="my-6 font-bold">
+      <div class="flex">
+        <span class="material-icons mr-2">schedule</span>
+        <p>
+          {formatDate(event.start)}
+          {formatTime(event.start)}
+        </p>
+      </div>
+      <div class="flex">
+        <span class="material-icons mr-2">location_on</span>
+        <p>{event.place}</p>
+      </div>
+    </div>
+    {#if event.body}
+      <div class="mt-8 flex flex-col gap-4">
+        <PortableText components={{}} value={event.body} />
+      </div>
+    {/if}
+    {#if data.auth?.user}
+      {data.auth.user.name}
+      <button on:click={() => signOut({ callbackUrl: "/" })}>Logg ut</button>
+    {:else}
+      <button on:click={() => signIn("google")}>Logg inn</button>
+    {/if}
```

Comment:
Skal disse være her? Kanskje bedre om de er oppe i headeren, og om de ser ut som knapper😊

Reply:
La de bare der som proof of concept. Tenker de bare kan stå til vi får noe design? Header er en god ide, men trengs bare på event og ikke på blogg.

Legger til litt styling :)

Reply:
Skjønner! Er ikke så nøye akkurat nå nei, bare la de stå for min del 😄